### PR TITLE
Add soaktest subprogram

### PR DESCRIPTION
**Pull request by dpino**
Adds new lwAFTR subprogram: soaktest. Soaktest runs lwAFTR on a given input for a period of time (default 0.10 seconds), repeatedly. It's aimed to verify the system remains stable and there's no crashes.

The PR also adds a soaktest script so the test is part of CI testing. The test is run with and without VLAN packets and both normal and on-a-stick mode.

This PR depends on #442. It also includes #437 and #439 so CI can pass.

### CODE REVIEW

File: src/program/lwaftr/soaktest/README [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  -D, --duration
+                             Run for duration seconds.
+                             Default: 0.10 seconds.
+  --on-a-stick
+                             Run in on-a-stick mode.
+
+Soaktest tests lwAFTR app over a period of time to validate system
+behaviour under production use.
+
+It's aimed to detect problems such as memory leaks which may cause
```
Comment (teknico) [inline src/program/lwaftr/soaktest/README]: "... such as memory leaks which only manifest themselves after multiple start-stop cycles."

File: src/program/lwaftr/soaktest/soaktest.lua [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+   if #args ~= 3 then print("Wrong number of arguments: "..#args) show_usage(1) end
+   if not opts.duration then opts.duration = 0.10 end
+   return opts, args
+end
+
+function run (args)
+   local opts, args = parse_args(args)
+   local conf_file, inv4_pcap, inv6_pcap = unpack(args)
+
+   local duration = opts.duration or 1
```
Comment (teknico) [inline src/program/lwaftr/soaktest/soaktest.lua]: This line is useless, right?

**Review by teknico — APPROVED** [review]
Only reviewing commits 2034f5a and 6c34511, since the other ones are part of other PRs.

LGTM with small changes (but cannot go in before the PRs it depends on, obviously).

File: src/program/lwaftr/soaktest/soaktest.lua (hunk 2 above)
Comment (dpino) [reply src/program/lwaftr/soaktest/soaktest.lua]: That's correct I initialized it to a default value twice, here and in parse_args.

**Pull request merged**